### PR TITLE
New folders added

### PR DESCRIPTION
**Pull request by Habinaya-K**

### CODE REVIEW

File: src/main/java/com/akhil/app/controllers/HealthController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@RestController
+public class HealthController {
+
+    @GetMapping("/healthup")
+    public String healthUp(){
+        return ("{\"Status\":\"Up\"}");
+    }
+
+    @GetMapping("/healthdown")
```
Comment (akhilyannabathula) [inline src/main/java/com/akhil/app/controllers/HealthController.java]: Health down is not necessary, if the healtUp is not responding it means health is down

File: src/main/java/com/akhil/app/controllers/HealthController.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.akhil.app.controllers;
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthController {
+
+    @GetMapping("/healthup")
```
Comment (akhilyannabathula) [inline src/main/java/com/akhil/app/controllers/HealthController.java]: /health is enough,
It should send status:Up if database server or any other services are working fine
It should send status:Down 
If database servers are down

As we don't have db servers here you can send status:Up directly

**Review by akhilyannabathula — CHANGES_REQUESTED** [review]

File: src/main/java/com/akhil/app/controllers/HealthController.java (hunk 2 above)
Comment (Habinaya-K) [reply src/main/java/com/akhil/app/controllers/HealthController.java]: Made the changes and committed again, please check

File: src/main/java/com/akhil/app/controllers/HealthController.java (hunk 1 above)
Comment (Habinaya-K) [reply src/main/java/com/akhil/app/controllers/HealthController.java]: Made changes, thanks

**Pull request merged**